cli/pkg/gh/gh.go

```diff
@@ -181,7 +181,7 @@ func labelRequest(rpo string, prNum int, labels []string) ([]Label, error) {
 	return resp, nil
 }
 
-func PreviewPr(rpo, dir string, pr *PullRequest) {
```

Comment:
Since all the preview does is print the PR info, there is no need to pass the pointer.